helper/communicator/step_connect.go

```diff
@@ -53,6 +53,7 @@ func (s *StepConnect) Run(state multistep.StateBag) multistep.StepAction {
 			Config:      s.Config,
 			Host:        s.Host,
 			WinRMConfig: s.WinRMConfig,
+			WinRMPort:   s.SSHPort,
```

Comment:
@cbednarski
```I haven't looked at this very closely yet, but why does this use SSHPort? Is that coming from the config template? This might be correct code but the name looks out of place.```

@pecigonzalo
```It does indeed look wrong, but most of the communicator implementation reuses components of the SSH one, like the SSH step that creates the NAT on virtualbox.```

@cbednarski
```The docs indicate there's a winrm_port option (https://packer.io/docs/templates/communicator.html#winrm_port) which is probably what we want here. I'm not sure about the NAT case that @markhuber mentioned.```